cmd/kola/options.go

```diff
@@ -227,6 +227,7 @@ func init() {
 	sv(&kola.QEMUOptions.BIOSImage, "qemu-bios", "", "BIOS to use for QEMU vm")
 	bv(&kola.QEMUOptions.UseVanillaImage, "qemu-skip-mangle", false, "don't modify CL disk image to capture console log")
 	sv(&kola.QEMUOptions.ExtraBaseDiskSize, "qemu-grow-base-disk-by", "", "grow base disk by the given size in bytes, following optional 1024-based suffixes are allowed: b (ignored), k, K, M, G, T")
+	bv(&kola.QEMUOptions.EnableTPM, "qemu-tpm", false, "enable TPM device in QEMU. Requires installing swtpm.")
```

Comment:
That's a bit confusing because the option also shows up for `kola run` but we don't want it for `kola run`. Can you move this into `cmd/kola/spawn.go`?

Reply:
i could, but is it worth the complexity? if someone passes this to `kola run` then they just get a tpm in every test case, shouldn't be problematic (eg. all GCP instances have a tpm)

Reply:
Ok to leave it here if we add more info
```suggestion
	bv(&kola.QEMUOptions.EnableTPM, "qemu-tpm", false, "enable TPM device in QEMU. Requires installing swtpm. This should only be set for 'kola spawn', and for 'kola run' the test case is responsible for creating a VM with a TPM explicitly")
```

Reply:
ok added a comment like that